submissions/anthoniaAdollo/html-forms/Contact form.html

```diff
@@ -63,18 +61,14 @@
         <li><a href="https://www.instagram.com">Instagram</a></li>
         <li><a href="https://www.facebook.com">Facebook</a></li>
     </ul>
-
 <footer>
     <nav>
-        <a href="#">Home</a>
-        <a href="#"> New Stock</a>
-        <a href="#">Sign Up</a>
-        <a href="#">Sale</a>
+        <a href="/Contact form.html">Contact Us</a>
```

Comment:
Hello, kindly ensure that your file names do not have spaces

Reply:
Thank you, I have made the recommended adjustments.